bgpd/bgp_open.h

```diff
@@ -57,6 +57,24 @@ struct graceful_restart_af {
 #define CAPABILITY_CODE_ROLE_LEN        1
 #define CAPABILITY_CODE_SOFT_VERSION_LEN 1
 
+static const struct message capcode_str[] = {
```

Comment:
hmm - so this will embed a copy in lots of modules, won't it? would you consider leaving the definition in one module and just putting a prototype/forward ref in a header so you can use it from bgp_packet.c ?

Reply:
Thanks, updated :)